### PR TITLE
Genevieve Schreiber signoff camera communication

### PR DESCRIPTION
**Pull request by TycheThePanda**
Requesting sign off for camera communications.

### CODE REVIEW

**Comment by TycheThePanda** [comment]
This branch is a different communications subsystem from the arduino communications subsystem.

**Review by JesseTNRoberts — COMMENTED** [review]
Schematic is insufficient.

File: Documentation/Signoffs/detail_design_camera_comms.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+ ![The distance measurement between cameras and the building](../Images/distanceparkinglot.png)
+
+3. Using an aggressive estimate to approximate the bandwidth required by the device, using 3 megapixels and 30 frames per second at the highest quality in the calculations, the camera would require only 76.8 Mbps[2], compared to the 300Mbps provided by the adapter, this is well within scope.
+4. The setup of this device would be simple, consisting of connecting the ethernet cable to a POE injector that feeds into the camera’s CAT5 RJ45 port directly. To power it, the device can be fed 5V-15V DC to its DC 2.0 power hole. The power consumption of this device is considered “low” by most standards and will be able to be designed for in the power subsystem.
+![Wiring diagram for the networking device](../Images/Camera_Comm_wiring.png)
+6. Thorough research showed this to be the only device currently on the market that provided the functionality necessary as well as fulfilling all of the requirements of the system.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/detail_design_camera_comms.md]: Since this is in your analysis section, I'm going to insist that you prove this claim.

**Review by JesseTNRoberts — COMMENTED** [review]
The signoff does not have the required structure and the schematic is not buildable.

**Review by JesseTNRoberts — COMMENTED** [review]
The electrical schematic is not complete unless this has a single wire which comes from it for power. 

Secondly, the schematic does not convey how you intend to connect this to the temp housing. There is nothing drawn except a highlighted area in the box.

**Review by JesseTNRoberts — COMMENTED** [review]
did not address feedback.

**Review by JesseTNRoberts — COMMENTED** [review]
So, the power cord plugs into nothing?

**Review by JesseTNRoberts — APPROVED** [review]

**Pull request merged**